packages/platform/src/services/PlayService.ts

```diff
@@ -609,12 +609,14 @@ export async function addCountdown(args, ctx: Context) {
 
   if (!currentGame?.activePeriod?.activeSegment) return null
 
+  const countdownDurationMs = args.seconds * 1000
   await ctx.prisma.periodSegment.update({
     where: {
       id: currentGame.activePeriod.activeSegment.id,
     },
     data: {
-      countdownExpiresAt: new Date(Date.now() + args.seconds * 1000),
+      countdownExpiresAt: new Date(Date.now() + countdownDurationMs),
```

Comment:
do this computation with dayjs (like dayjs().add(countdownDurationMs, 'ms') or something like that

Reply:
I still needed to convert this into a Date - it did not work with toString(). Maybe that has something to do with prisma with the DateTime type